Backpack/Rating/Classes/BPKRatingTextWrapper.m

```diff
@@ -100,48 +99,72 @@ - (void)setSize:(BPKRatingSize)size {
     }
 }
 
-- (void)updateFontSizes {
-    switch (self.size) {
-        case BPKRatingSizeLarge:
-            self.titleLabel.fontStyle = BPKFontStyleTextLgEmphasized;
-            self.subtitleLabel.fontStyle = BPKFontStyleTextBase;
-            break;
-        case BPKRatingSizeBase:
-            self.titleLabel.fontStyle = BPKFontStyleTextBaseEmphasized;
-            self.subtitleLabel.fontStyle = BPKFontStyleTextSm;
-            break;
-        case BPKRatingSizeSmall:
-            self.titleLabel.fontStyle = BPKFontStyleTextSmEmphasized;
-            self.subtitleLabel.fontStyle = BPKFontStyleTextXs;
-            break;
-        case BPKRatingSizeExtraSmall:
-            self.titleLabel.fontStyle = BPKFontStyleTextXsEmphasized;
-            self.subtitleLabel.fontStyle = BPKFontStyleTextXs;
-            break;
-    }
-}
-
 - (void)setLayout:(BPKRatingLayout)layout {
     BPKAssertMainThread();
     if (_layout != layout) {
         _layout = layout;
 
+        [self updateFontSizes];
         [self updateConstraints];
```

Comment:
This shouldn't be called directly, instead we should call `setNeedsUpdateConstraints`

Reply:
I don't think I ever actually meant to override/call the UIKit method - I think I just chose a name that clashed.
Have renamed so that we can avoid invoking it ourselves.

Reply:
I actually think this is a good situation to be using Apple's method though 🤔

Reply:
Oh haha - I'll use that then 👍